### PR TITLE
Skeletons

### PR DESCRIPTION
**Pull request by tymnik**
- skeletons added;
- small fixes;
- animation for searchbar, hovers for some elements;

### CODE REVIEW

File: src/components/skeletons/AboutUsPageSkeleton/AboutUsPageSkeleton.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          <li>
+            <Skeleton className={styles.devCard} />
+          </li>
+          <li>
+            <Skeleton className={styles.devCard} />
+          </li>
+          <li>
+            <Skeleton className={styles.devCard} />
+          </li>
+        </ul>
```
Comment (VovaLyapota) [inline src/components/skeletons/AboutUsPageSkeleton/AboutUsPageSkeleton.tsx:36]: it must be done with map() method

File: src/components/skeletons/ActivityPageSkeleton/ActivityPageSkeleton.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+          <li>
+            <Skeleton className={styles.block} />
+          </li>
+          <li>
+            <Skeleton className={styles.block} />
+          </li>
+          <li>
+            <Skeleton className={styles.block} />
+          </li>
+        </ul>
```
Comment (VovaLyapota) [inline src/components/skeletons/ActivityPageSkeleton/ActivityPageSkeleton.tsx:21]: map

File: src/components/skeletons/ProfilePageSkeleton/ProfilePageSkeleton.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+          <li>
+            <Skeleton className={styles.input} />
+          </li>
+          <li>
+            <Skeleton className={styles.input} />
+          </li>
+          <li>
+            <Skeleton className={styles.input} />
+          </li>
+        </ul>
```
Comment (VovaLyapota) [inline src/components/skeletons/ProfilePageSkeleton/ProfilePageSkeleton.tsx:21]: map

File: src/components/skeletons/RegisterPageSkeleton/RegisterPageSkeleton.tsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+            <li>
+              <Skeleton className={styles.input} />
+            </li>
+            <li>
+              <Skeleton className={styles.input} />
+            </li>
+            <li>
+              <Skeleton className={styles.input} />
+            </li>
+          </ul>
```
Comment (VovaLyapota) [inline src/components/skeletons/RegisterPageSkeleton/RegisterPageSkeleton.tsx:22]: map

File: src/components/skeletons/RegisterPageSkeleton/RegisterPageSkeleton.tsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+        <li>
+          <Skeleton className={styles.stepCardOne} />
+        </li>
+        <li>
+          <Skeleton className={styles.stepCardTwo} />
+        </li>
+        <li>
+          <Skeleton className={styles.stepCardThree} />
+        </li>
+      </ul>
```
Comment (VovaLyapota) [inline src/components/skeletons/RegisterPageSkeleton/RegisterPageSkeleton.tsx:38]: map

File: src/components/skeletons/RoadmapPageSkeleton/RoadmapPageSkeleton.tsx [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+          <li>
+            <Skeleton className={styles.blockSection} />
+          </li>
+          <li>
+            <Skeleton className={styles.blockSection} />
+          </li>
+          <li>
+            <Skeleton className={styles.blockSection} />
+          </li>
+        </ul>
```
Comment (VovaLyapota) [inline src/components/skeletons/RoadmapPageSkeleton/RoadmapPageSkeleton.tsx:35]: map

File: src/components/skeletons/RoadmapPageSkeleton/RoadmapPageSkeleton.tsx [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+          <li>
+            <Skeleton className={styles.textBlock} />
+          </li>
+          <li>
+            <Skeleton className={styles.textBlock} />
+          </li>
+          <li>
+            <Skeleton className={styles.textBlock} />
+          </li>
+        </ul>
```
Comment (VovaLyapota) [inline src/components/skeletons/RoadmapPageSkeleton/RoadmapPageSkeleton.tsx:48]: map

File: src/components/skeletons/RoadmapsPageSkeleton/RoadmapsPageSkeleton.tsx [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+            <li>
+              <Skeleton className={styles.addBlock} />
+            </li>
+            <li>
+              <Skeleton className={styles.addBlock} />
+            </li>
+            <li>
+              <Skeleton className={styles.addBlock} />
+            </li>
+          </ul>
```
Comment (VovaLyapota) [inline src/components/skeletons/RoadmapsPageSkeleton/RoadmapsPageSkeleton.tsx:23]: map

File: src/components/skeletons/RoadmapsPageSkeleton/RoadmapsPageSkeleton.tsx [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+        </div>
+        <ul className={styles.images}>
+          <li className={styles.firstImage}>
+            <Skeleton className={styles.image} />
+          </li>
+          <li className={styles.secondImage}>
+            <Skeleton className={styles.image} />
+          </li>
+        </ul>
```
Comment (VovaLyapota) [inline src/components/skeletons/RoadmapsPageSkeleton/RoadmapsPageSkeleton.tsx:32]: map

File: src/pages/RegistrationPage/RegistrationPage.tsx [hunk 10]
```diff
@@ -13,34 +15,41 @@ import PageTitle from 'components/ui/PageTitle';
...
+                >
+                  LOG IN
+                </Link>
+              </p>
+            </div>
+          </div>
+          <RoadPlanning />
+        </Container>
+      )}
+    </>
```
Comment (VovaLyapota) [inline src/pages/RegistrationPage/RegistrationPage.tsx]: you don't need react fragment here

File: src/pages/RoadmapPage/RoadmapPage.tsx [hunk 11]
```diff
@@ -1,6 +1,7 @@
 import { useEffect } from 'react';
 import { useParams } from 'react-router';
 
+import RoadmapPageSkeleton from 'src/components/skeletons/RoadmapPageSkeleton';
```
Comment (VovaLyapota) [inline src/pages/RoadmapPage/RoadmapPage.tsx]: src/

File: src/components/skeletons/AboutUsPageSkeleton/AboutUsPageSkeleton.tsx (hunk 1 above)
Comment (tymnik) [reply src/components/skeletons/AboutUsPageSkeleton/AboutUsPageSkeleton.tsx:36]: done

File: src/components/skeletons/ActivityPageSkeleton/ActivityPageSkeleton.tsx (hunk 2 above)
Comment (tymnik) [reply src/components/skeletons/ActivityPageSkeleton/ActivityPageSkeleton.tsx:21]: done

File: src/components/skeletons/ProfilePageSkeleton/ProfilePageSkeleton.tsx (hunk 3 above)
Comment (tymnik) [reply src/components/skeletons/ProfilePageSkeleton/ProfilePageSkeleton.tsx:21]: done

File: src/components/skeletons/RegisterPageSkeleton/RegisterPageSkeleton.tsx (hunk 4 above)
Comment (tymnik) [reply src/components/skeletons/RegisterPageSkeleton/RegisterPageSkeleton.tsx:22]: done

File: src/components/skeletons/RegisterPageSkeleton/RegisterPageSkeleton.tsx (hunk 5 above)
Comment (tymnik) [reply src/components/skeletons/RegisterPageSkeleton/RegisterPageSkeleton.tsx:38]: it is easier to style each card in this way as they all are different.

File: src/components/skeletons/RoadmapPageSkeleton/RoadmapPageSkeleton.tsx (hunk 6 above)
Comment (tymnik) [reply src/components/skeletons/RoadmapPageSkeleton/RoadmapPageSkeleton.tsx:35]: done

File: src/components/skeletons/RoadmapPageSkeleton/RoadmapPageSkeleton.tsx (hunk 7 above)
Comment (tymnik) [reply src/components/skeletons/RoadmapPageSkeleton/RoadmapPageSkeleton.tsx:48]: done

**Review by Tanyapavliuk — APPROVED** [review]

**Review by VovaLyapota — APPROVED** [review]

**Pull request merged**